backend/src/apiserver/resource/resource_manager.go

```diff
@@ -348,14 +353,14 @@ func (r *ResourceManager) RetryRun(runId string) error {
 		return util.NewInternalServerError(err, "Failed to retrieve the runtime pipeline spec from the run")
 	}
 
-	newWorkflow, podsToDelete, err := formulateRetryWorkflow(&workflow)
+	newWorkflow, _, err := formulateRetryWorkflow(&workflow)
 	if err != nil {
 		return util.Wrap(err, "Retry run failed.")
 	}
-
-	if err = deletePods(podsToDelete, newWorkflow.ObjectMeta.Namespace); err != nil {
-		return util.NewInternalServerError(err, "Retry run failed. Failed to clean up the failed pods from previous run.")
-	}
+	//
```

Comment:
Commented code?

Reply:
sorry it was not ready yet. it's ready now